### PR TITLE
CRUD pages

### LINKED ISSUE #23: UI - Improve pages style

### PR DESCRIPTION
**Pull request by Kellane**
Closes #23

### CODE REVIEW

**Review by mateusKoppe — APPROVED** [review]
Nice!

**Review by arufonsekun — APPROVED** [review]
clapping-leonardo-dicaprio

File: client/src/pages/Pages.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <ListGroup variant="flush">
+              <ListGroup.Item action href={`/pages/${page.id}`} key={page.id}>
+                {page.title}
+              </ListGroup.Item>
+          </ListGroup>             
+            <Button size="sm" variant="danger" onClick={() => deletePage(page.id)}>
+              Delete
+            </Button>
+          </div>
+        )})}
```
Comment (jeanchilger) [inline client/src/pages/Pages.jsx:26]: Change the spacing:

```suggestion
        {pages.map((page) => {
          return (
            <div key={page.id}>
              <ListGroup variant="flush">
                <ListGroup.Item action href={`/pages/${page.id}`} key={page.id}>
                  {page.title}
                </ListGroup.Item>
              </ListGroup>             
              <Button size="sm" variant="danger" onClick={() => deletePage(page.id)}>
                Delete
              </Button>
            </div>
        )})}
```

File: client/src/hooks/usePages.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    const response = await Api.post(`/pages`, page);
+    const newPage = response.data;
+    setPages((pages) => [...pages, newPage]);
+  };
+
+  const updatePage = async (id, data) => {
+    const index = pages.findIndex((page) => page.id === id);
+    const newPage = { ...pages[index], ...data };
+
+
```
Comment (jeanchilger) [inline client/src/hooks/usePages.js:24]: Extra line here:

```suggestion
```

**Review by jeanchilger — CHANGES_REQUESTED** [review]
I've added some code formatting suggestions. Everything seems to be working as expected.

**Review by jeanchilger — APPROVED** [review]

**Pull request merged**